### PR TITLE
remove some unused collectors

### PR DESCRIPTION
**Pull request by krasi-georgiev**
cc @LiliC , @brancz 
Signed-off-by: Krasi Georgiev <8903888+krasi-georgiev@users.noreply.github.com>

### CODE REVIEW

**Comment by LiliC** [comment]
cc @pgier do you mind having a look as well, as you know node-exporter best.

**Comment by s-urbaniak** [comment]
sounds good to me, leaving a final pass to @brancz as we are removing quite some collectors.

File: jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet [hunk 1]
```diff
@@ -89,7 +89,19 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           '--path.procfs=/host/proc',
           '--path.sysfs=/host/sys',
           '--path.rootfs=/host/root',
-
+          '--no-collector.wifi',
+          '--no-collector.hwmon',
+          '--no-collector.netstat',
+          '--no-collector.sockstat',
+          '--no-collector.entropy',
```
Comment (paulfantom) [inline jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet]: This one is quite important when doing post-mortems of DDoS. Especially when entropy can be quickly exhausted.

File: jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet [hunk 2]
```diff
@@ -89,7 +89,19 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           '--path.procfs=/host/proc',
           '--path.sysfs=/host/sys',
           '--path.rootfs=/host/root',
-
+          '--no-collector.wifi',
+          '--no-collector.hwmon',
+          '--no-collector.netstat',
```
Comment (paulfantom) [inline jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet]: This one and conntrack needs some thought as those might affect debuggability of networking issues.

File: jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet [hunk 3]
```diff
@@ -89,7 +89,19 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
...
+          '--no-collector.sockstat',
+          '--no-collector.entropy',
+          '--no-collector.filefd',
+          '--no-collector.time',
+          '--no-collector.conntrack',
+          '--no-collector.bcache',
+          '--no-collector.stat',
+          '--no-collector.xfs',
+          '--no-collector.pressure',
```
Comment (paulfantom) [inline jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet]: It is providing CPU and memory pressure mechanism information and I am not sure if those are not reflecting k8s scheduler states.

**Comment by krasi-georgiev** [comment]
updater to disable only hwmon

The big wins are not in the node_exporter anyway so not worth spending more time here.

**Pull request merged**